linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java

```diff
@@ -222,25 +222,14 @@ public void testBuildModelWithSnapshotBom_JdkVersionActivation()
   }
 
   @Test
-  public void testFindLocalArtifacts_currentDirectory() {
+  public void testFindLocalArtifacts() {
     // Current working directory of linkage-monitor should have one linkage monitor artifact
     ImmutableMap<String, String> localArtifacts =
-        LinkageMonitor.findLocalArtifacts(system, session, Paths.get("."));
+        LinkageMonitor.findLocalArtifacts(
+            system, session, Paths.get("src/test/resources/testproject"));
 
-    Truth.assertThat(localArtifacts).hasSize(1);
-    Truth.assertThat(localArtifacts).containsKey("com.google.cloud.tools:linkage-monitor");
-  }
-
-  @Test
-  public void testFindLocalArtifacts_RootDirectory() {
-    // Root of cloud-opensource-java has more than 10 pom.xml files
-    ImmutableMap<String, String> localArtifacts =
-        LinkageMonitor.findLocalArtifacts(system, session, Paths.get(".."));
```

Comment:
This test was reading all `pom.xml` files under cloud-opensource-java directory.